src/router/metadata/cap.template.ts

```diff
@@ -3,21 +3,26 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import { FhirVersion } from 'fhir-works-on-aws-interface';
+import { FhirVersion, ProductInfo } from 'fhir-works-on-aws-interface';
 
-export default function makeStatement(rest: any, orgName: string, url: string, fhirVersion: FhirVersion) {
+export default function makeStatement(rest: any, productInfo: ProductInfo, url: string, fhirVersion: FhirVersion) {
     const cap: any = {
         resourceType: 'CapabilityStatement',
+        name: productInfo.productMachineName ?? 'FhirServerCapabilityStatement',
+        title: `${productInfo.productTitle ?? 'Fhir Server'} Capability Statement`,
+        description: productInfo.productDescription ?? `A FHIR ${fhirVersion} Server Capability Statement`,
+        purpose: productInfo.productPurpose ?? `A statement of this system's capabilities`,
+        copyright: productInfo.copyright ?? ' ',

```

Comment:
nitpick: Could this just be `''` instead of a blank space `' '`?

Reply:
Thanks, fixed!

Reply:
Interesting looks like the unit tests are failing due to copyright being an empty string: `InvalidResourceError: data.copyright should match pattern "^[ \r\n\t\S]+$"`

Reply:
Could we change copyright to be conditionally added (i.e. only have the `copyright` param if it is present)